packages/lumx-react/src/components/thumbnail/Thumbnail.tsx

```diff
@@ -99,13 +117,17 @@ const CLASSNAME: string = getRootClassName(COMPONENT_NAME);
  */
 const DEFAULT_PROPS: DefaultPropsType = {
     align: Alignment.left,
+    enableCrossOrigin: true,
```

Comment:
To avoid side effects, should it be set to false by default?
What is the current behavior?

Reply:
@radiuso I think we need it enabled by default for now because we are alreday using it heavily in the product and we would have to rewrite a lot of things.

Reply:
Yup, validated with SA team as well

Reply:
We want this one enable by default, if not pictures aren't displayed into Lumapps :)